content/20.treatments.md

```diff
@@ -86,6 +86,68 @@ A list of current studies and their results, using carefully the information req
+Correction for multiple primary outcomes can be adjusted post-hoc by recalculating p-values, but all other issues are design and statistical problems that cannot be corrected for.
+Additionally, the issue of drastic disparities between pre-registration and published protocol may be suggestive of p-hacking.
+Again, conclusions cannot de generalized outside of the study, but the results support further investigation.
+
+A randomized trial from the Shanghai Public Health Clinical Center of 30 COVID-19 patients studied whether HCQ increased rates of virological clearance by respiratory pharyngeal swab on Day 7 post-treatment compared to standard care [how to cite? http://www.zjujournals.com/med/CN/10.3785/j.issn.1008-9292.2020.03.03].
+This trial was published in Chinese with an abstract also in English.
+Only the English abstract was read and interpreted.
+The trial found HCQ showed comparable outcomes to standard care with regard to virological clearance rate, time to virological clearance, and time to body temperature normalization.
+A known weakness is small sample size.
+This problem suggests the study is underpowered to detect potentially useful differences and precludes interpretation of results.
+Additionally, because only the abstract could be read, other design and analysis issues could be present.
+Conclusions should be interpreted very cautiously.
+However, these preliminary negative results do reiterate the need for further study.
+
+A case study treated 11 consecutive patients with HCQ + azithromycin using the same dosing regime reported by Gautret et al [@doi:10.1016/j.medmal.2020.03.006].
+One patient died, two were transferred to the ICU, and one developed a prolonged QT interval leading to discontinuation of HCQ + azithromycin.
+As in the Gautret et al study, virological clearance was assessed at Day 6 post-treatment by nasopharyngeal swabs.
+Of the ten living patients at the time, eight of the patients remained positive for SARS-CoV-2 RNA.
+Interpretation of conclusions are severely limited by lack of comparison group and small sample size.
+However, these results stand in contrast to claims by Gautret et al and illustrates the need for better and further investigation.
+
+There is also conversation of clinical trials showing HCQ is superior to control treatment in inhibiting the exacerbation of pneumonia.
+These conversations cite a Letter to the Editor that makes this claim and refers to 15 pre-registration identifiers from the Chinese Clinical Trial Registry [@doi:10.5582/bst.2020.01047].
+When these identifiers are followed back to the registry, most trials claim to still be recruiting patients and no trials have uploaded data or publications.
+
+###### Summary
+
```

Comment:
What do you think about adding a section -- either as part of the summary, or before/after it, like "Summary and Further Avenues" or something -- talking about what future research regarding hydroxychloroquine and its derivatives could/should explore? (Unless there is a section elsewhere on this?)

I found this medicinal chemistry paper on hydroxychloroquine alternatives (hydroxyferroquine derivatives, which are organometallic agents) that showed selectivity against SARS-CoV back in 2006. Links here:
https://www.ncbi.nlm.nih.gov/pubmed/16640347
https://pubs.acs.org/doi/full/10.1021/jm0601856#

A section on further research on these antivirals could summarize:

What information we still need to know about hydroxychloroquine and its effectiveness / what current data are lacking (which you do here already)
Alternatives to hydroxychloroquine as antivirals that might be worth exploring (^ paper above)
Maybe... some notes on how antivirals are developed and where we could go from here? There might be another section on this part, not sure!
Also found this paper on antiviral development, which may or may not be worth citing (but putting it here to remind myself and anyone else interested, to think about):
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4765712/
https://www.elsevier.es/es-revista-medicina-universitaria-304-articulo-history-progress-antiviral-drugs-from-S166557961500037X
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2972676/pdf/main.pdf

Reply:
I think that could be interesting and useful, but I don't think I know enough about the overall vision for the paper to comment. @rando2, perhaps you could weigh in?

Reply:
Makes sense. If we'd rather not add a whole section, a line or two at either 150 or 139 might be interesting just to note HCQ alternatives. For example, "Given the great focus in investigating HCQ as a potential antiviral treatment for SARS-CoV-2, it may be of interest to researchers to explore related alternatives. For example, hydroxyferroquine derivatives of HCQ have been described as a class of bioorganometallic compounds that exert antiviral effects with some selectivity for SARS-CoV."

Reply:
Sorry I missed this, @hufengling and @dianerafi ! This sounds completely in line with the vision for the paper. In terms of sections, I would say either here or we could potentially add a section at the end of Therapeutics where we have people suggest different avenues that haven't been as widely explored.
I think what I'll do for now is pull this out into a separate issue, and that way we can make sure it gets integrated and loop in the people who wrote the original outline for this section to see if they have any thoughts too!

Reply:
Thanks @rando2, that sounds great! I'll look over at the new issue now.

Reply:
I think it would be great for the summary to include the study design of current trials in the summary. UPenn itself has the PATCH (Prevention and Treatment of COVID-10 with Hydroxychloroquine) trial starting soon and then there is another multi-institutional trial for in-patient population.